### ISSUE TITLE
Slideshow thumbnails + dialog image missing alt text / empty links

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1772]
Example:
https://exhibits.stanford.edu/maps-of-africa/feature/cape-of-good-hope?page=8

**Issues**
1. the thumbnails are missing alt text
2. the thumbnails are links so there is also no link text
3. the link opens a full-size (or "bigger") image that has no alt text but does have a caption

**thumbnails**
- [x] add the image title as alt text to thumbnails - solves 1 & 2

**full-size image** 
- [x] add nul `alt=""` to the image (caption is sufficient) - solves 3
